geonode/base/api/tests.py

```diff
@@ -136,6 +136,96 @@ def test_gropus_list(self):
             pub_invite_1.delete()
             pub_invite_2.delete()
 
+    def test_create_group(self):
+        """
+        Ensure only Admins can create groups.
+        """
+        data = {
+            "title": "group title",
+            "group": 1,
+            "slug": "group_title",
+            "description": "test",
+            "access": "private",
+            "categories": []
+            }
+        try:
+            # Anonymous
+            url = reverse('group-profiles-list')
+            response = self.client.post(url, data=data, format='json')
+            self.assertEqual(response.status_code, 403)
+
+            # Registered member
+            self.assertTrue(self.client.login(username='bobby', password='bob'))
+            response = self.client.post(url, data=data, format='json')
+            self.assertEqual(response.status_code, 403)
+
```

Comment:
To be 100% compatible with the issue requirements is a good option to have here also the test for the group admin